src/views/Peers.tsx

```diff
@@ -20,21 +20,24 @@ import {
     RadioChangeEvent,
     Dropdown,
     Menu,
-    Alert, Select, Modal, Button, message
+    Alert, Select, Modal, Button, message, Popover
 } from "antd";
 import {Peer} from "../store/peer/types";
 import {filter} from "lodash"
 import {formatOS, timeAgo} from "../utils/common";
 import {ExclamationCircleOutlined} from "@ant-design/icons";
 import ButtonCopyMessage from "../components/ButtonCopyMessage";
-import {Group} from "../store/group/types";
+import {Group, GroupPeer} from "../store/group/types";
+import PeerGroupsUpdate from "../components/PeerGroupsUpdate";
 
 const { Title, Paragraph } = Typography;
 const { Column } = Table;
 const { confirm } = Modal;
 
 interface PeerDataTable extends Peer {
-    key: string
+    key: string;
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `··`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  key: string;
```